src/Util/ProxyUtils.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace Friendica\Util;
+
+use Friendica\BaseModule;
+use Friendica\BaseObject;
+use Friendica\Core\Config;
+use Friendica\Core\System;
+
+require_once 'include/security.php';
+
+/**
+ * @brief Proxy utilities
+ */
+class ProxyUtils
```

Comment:
I'm still not happy about this class name. If you won't change it to `Friendica\Util\Proxy` and only use `ProxyUtils` as an alias in `src/Module/Proxy.php`, I'll make the change myself.

Reply:
Hmm. :-/ Okay, I learned this from Java language. ;-)